scrapers/data18.yml

```diff
@@ -2,34 +2,24 @@ name: data18
 sceneByURL:
   - action: scrapeXPath
     url:
-      - data18.com/content
+      - data18.com/scenes
     scraper: sceneScraper
 xPathScrapers:
   sceneScraper:
     common:
-      $performer: //p[b[text()='Starring:']]/a
-      $studio: //p[b[text()='Site:' or text()='Studio:']]/a
+      $performer: //b[text()='Pornstars / Cast']/../..//a[@class='bold gen']
+      $studio: (//b[text()="Studio"] | //b[text()="Network"])/../a[last()]
     scene:
-      Title: //h1/text()
+      Title: //span/following-sibling::h1/a/text()
```

Comment:
are there scenes with more than one `h1` ? (did a test in a few and `//h1/a/text()` seemed enough)